src/cubic_loader/qlik/rds_utils.py

```diff
@@ -0,0 +1,170 @@
+from typing import List
+from typing import Optional
+from datetime import date
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+from cubic_loader.utils.remote_locations import ODS_SCHEMA
+from cubic_loader.qlik.utils import DFMSchemaFields
+
+
+# pylint: disable=too-many-branches
+def qlik_type_to_pg(qlik_type: str, scale: int) -> str:
+    """
+    convert qlik datatype from DFM file to postgres type
+
+    :param qlik_type: QLIK data type from DFM file
+    :param scale: max number of digits to right of decimal
+
+    :return: postgres type
```

Comment:
Helper function to convert QLIK datatypes (indicated by .dfm files) to postgres datatypes per this reference: https://help.qlik.com/en-US/replicate/May2022/Content/Replicate/Main/Amazon%20S3/ar_mapping_to_dfm_s3.htm

Using these datatypes will optimize postgres queries and table storage space, vs saving everything as strings.